### PR TITLE
Issue 280: sustainability child pages - Biodiversity 

### LINKED ISSUE #280: Page: sustainability child pages
Migrating: https://www.sunstar.com/sustainability/environment/biodiversity/
https://www.sunstar.com/sustainability/environment/products/

### PR DESCRIPTION
**Pull request by shrotia**
Fixes #280 

Test URLs:
- Original: https://www.sunstar.com/sustainability/environment/biodiversity/
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability/environment/biodiversity
- After: https://issue-280--sunstar--hlxsites.hlx.live/sustainability/environment/biodiversity

- [x] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------

-  Introduced a new column variant for Row Cards, allowing the display of long cards with Image & Text side by side.  | [doc-link](https://issue-280--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/cards&index=3)
- Added variant for Heading Divider within columns.  | [doc-link](https://issue-280--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/columns&index=19)
- Added Extra Muted variant at Text Block.  | [doc-link](https://issue-280--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/text-image&index=6)

### CODE REVIEW

**Comment by JiangLong2019** [comment]
Hey @shrotia , not sure if this PR is under review, I just have a few comments after looking the EDS page. If you are already aware of these, please ignore.

1. In mobile view, the order of text and image is different from original site.
Original site:
image

EDS site:
image

2. Nit: the image is not in the same line as the title, looks a little bit strange.
image

**Comment by shrotia** [comment]
> Hey @shrotia , not sure if this PR is under review, I just have a few comments after looking the EDS page. If you are already aware of these, please ignore.
> 
> 1. In mobile view, the order of text and image is different from original site.
>    Original site:
> 
> image
> EDS site: image
> 
> 2. Nit: the image is not in the same line as the title, looks a little bit strange.
> 
> image

@JiangLong2019 yes, it is in review. Thanks for the feedback, yes I completely overlooked the first. let me address both.

File: blocks/text/text.css [hunk 1]
```diff
@@ -58,6 +58,12 @@ main .block.text.muted {
   padding-bottom: 0.5rem;
 }
 
+main .block.text.extra-muted, main .block.text.extra-muted p {
+  font-size: var(--body-font-size-xs);
```
Comment (mkbansal1) [inline blocks/text/text.css:62]: Its agreed to correct the muted version. I already had a card for same, for which PR is raised
https://github.com/hlxsites/sunstar/pull/328

extra-muted is not required.

Comment (shrotia) [reply blocks/text/text.css:62]: I will stick with the muted variant then

**Comment by shrotia** [comment]
> Nit: the image is not in the same line as the title, looks a little bit strange.

Adding this padding for `text-col` breaks the regular column layout [at healthy-thunking page](https://github.com/hlxsites/sunstar/commit/6bee5dd1dee990b441ebd1a616337b4070a508b3).
`.columns:not(.no-padding) > div > div:not(.columns-img-col) {
  padding: 1rem 0;
}`

I can see on healthy-thinking page, text-col with or without `.no-padding` selector require this 1rem padding.

I am thinking of modifying like this, though it looks ugly, or otherwise revisit healthy-thunking page,  what do you suggest?
`.columns:not(.absolute-no-padding) > div > div:not(.columns-img-col) div:not(.img-col) {
    padding: 1rem 0;
  }`

**Comment by shrotia** [comment]
@JiangLong2019 @mkbansal1 Created new PR : https://github.com/hlxsites/sunstar/pull/333 (please have a look)

**Pull request closed without merging**